examples/nffl-cli/README.md

```diff
@@ -0,0 +1,43 @@
+## How to build
+```sh
+npm install
```

Comment:
🤢 This is huge tech debt just for this; We should write CLIs in the teams native language, make a ticket to write it in rust

Reply:
shall we remove ti for a different PR?